### ISSUE TITLE
Add actions and reducers for note-taking to main Redux store

### ISSUE THREAD

**Issue by hopetambala** [CyberReboot/CRviz#380]
Create actions and reducers in tooltip-reducer that trigger on  "click" event in "Visualization.js"

**Comment by hopetambala** [comment]
tooltip-reducer
```
import { createAction, handleActions } from "redux-actions";


const defaultState = {
  position:[0,0]
};
// ACTIONS

const setPosition = createAction("POSITION");


const reducer = handleActions(
  {
    [setPosition]: (state, { action }) => {
      if( action.position !== 0 )
      {
      	const position = action.position;
      	return { ...state, position };
      }
      return { ...state };
    }
  },
  defaultState
);

const getPosition = (state) => state.position;

export default reducer;
export { setPosition, getPosition};
```

**Comment by hopetambala** [comment]
Changed to move abstract position from note-taking feature
```
#controls.js
import { createAction, handleActions } from "redux-actions";

const defaultState = {
  ...
  position: [0,0]
};

// ACTION CREATORS
...
const setPosition = createAction("POSITION");

// REDUCERS

const reducer = handleActions(
  {
    [setControls]: (state, { payload }) =>{
      ...
      const position = payload.position || state.position;
      return { 
        ...state,
        ...
        position: position
      }
    },
   ...
    [setPosition]: (state, { payload }) => ({ ...state, position: payload }),
  },
  defaultState
);

// SELECTORS
...
const getPosition = (state) => state.controls.position;

export default reducer;
export { ...d setPosition, getPosition };

```

**Comment by rashley-iqt** [comment]
Fixed by #415
